airtime_mvc/application/common/FileIO.php

```diff
@@ -70,11 +70,17 @@ public static function smartReadFile($filePath, $size, $mimeType)
             ob_end_flush();
         }
 
-        // NOTE: We can't use fseek here because it does not work with streams
-        // (a.k.a. Files stored in the cloud)
```

Comment:
I think this was specific to files coming from an S3 bucket on SaaS. The proper fix nowadays would most likely be to use a file system abstraction library like [Flysytem](http://flysystem.thephpleague.com/).